Content.Client/Stylesheets/Redux/SheetletConfigs/IButtonConfig.cs

```diff
@@ -0,0 +1,89 @@
+using System.Numerics;
+using Content.Client.Stylesheets.Redux.Stylesheets;
+using Robust.Client.Graphics;
+using Robust.Shared.Utility;
+
+namespace Content.Client.Stylesheets.Redux.SheetletConfigs;
+
+public interface IButtonConfig : ISheetletConfig
+{
+    public ResPath BaseButtonPath { get; }
+    public ResPath OpenLeftButtonPath { get; }
+    public ResPath OpenRightButtonPath { get; }
+    public ResPath OpenBothButtonPath { get; }
+    public ResPath SmallButtonPath { get; }
+    public ResPath RoundedButtonPath { get; }
+    public ResPath RoundedButtonBorderedPath { get; }
+
+    public ColorPalette ButtonPalette { get; }
+    public ColorPalette PositiveButtonPalette { get; }
+    public ColorPalette NegativeButtonPalette { get; }
+
+    public StyleBox ConfigureBaseButton(IStyleResources sheet)
```

Comment:
Yeah this really is not a good use of DIMs. I would rather these be extension methods or something.

Reply:
These should be on `ButtonSheetlet` anyway probably